src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -26,42 +26,77 @@
     @Autowired
     private BookRepository bookRepository;
 
+    /**
+     *
+     * @param name Name of a person
+     * @param model Model of the application
+     * @return "Hola" + the name passed as parameters
+     */
     @GetMapping("/greeting")
     public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
             Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
 
+    /**
+     *
+     * @return all books
```

Comment:
Suggestion:
I recommend using -> {@link Book} in all place where you name the word book.